### PR TITLE
Minor fixes (proposal)

### PR DESCRIPTION
**Pull request by kant**

### CODE REVIEW

File: docs/source/build_instructions.md [hunk 1]
```diff
@@ -82,9 +82,9 @@ If you don't have Administrator access, you can add a user-level PATH
 environment variable and put `C:\src\depot_tools` at the front, but
 if your system PATH has a Python in it, you will be out of luck.
 
-Also, add a DEPOT_TOOLS_WIN_TOOLCHAIN system variable in the same way, and set
+Also, add a `DEPOT_TOOLS_WIN_TOOLCHAIN` system variable in the same way, and set
 it to 0. This tells depot_tools to use your locally installed version of Visual
-Studio (by default, depot_tools will try to use a google-internal version).
+Studio (by default, depot_tools` will try to use a google-internal version).
```
Comment (kqyang) [inline docs/source/build_instructions.md]: Do you mean `depot_tools` here?

File: docs/source/build_instructions.md [hunk 2]
```diff
@@ -142,7 +142,7 @@ $ cd src
 #### Linux and Mac
 
 Shaka Packager uses [Ninja](https://ninja-build.org) as its main build tool,
-which is bundled in depot_tools.
+which is bundled in `depot_tools.
```
Comment (kqyang) [inline docs/source/build_instructions.md]: do you mean `depot_tools` here?

File: docs/source/build_instructions.md [hunk 3]
```diff
@@ -62,18 +62,18 @@ Download the
 [depot_tools bundle](https://storage.googleapis.com/chrome-infra/depot_tools.zip)
 and extract it somewhere.
 
-*** note
+***Note***
```
Comment (kqyang) [inline docs/source/build_instructions.md]: Thanks for fixing this. It is certainly better than the original code, but I am not sure if it is what we want.

What do you think of just replacing this line with just "***"?

**Comment by vaage** [comment]
@kant, Are you still working on this Pull Request?

**Comment by joeyparrish** [comment]
Closing due to lack of response.

File: docs/source/build_instructions.md (hunk 2 above)
Comment (kant) [reply docs/source/build_instructions.md:145]: you are right, `depot_tools` is the way to go (my minor proposal)

**Pull request closed without merging**